tests/PS01/readme.md

```diff
@@ -80,11 +80,14 @@ Not covered by the IDS file (must be checked otherwise):
         - one `.LENGTHUNIT.` and
         - one `.PLANEANGLEUNIT.`.
 - (RI-03) An `IfcGeometricRepresentationContext` shall be associated with the `IfcProject`. The project origin should be `(0,0,0)`.
+- (RI-64) The placement of `IfcAnnotation` shall be relative to the `IfcFacility` in which it is contained.
+- (RI-65) The placement of `IfcFacility` shall be absolute within the `IfcGeometricRepresentationContext`.
 - (RI-77) `IfcAnnotation.USERDEFINED` with `ObjectType='Annotation point'` shall have `'Point'` geometry and use `IfcCartesianPoint`.
-- (RI-87) The model shall containts an IfcProjectedCRS that defines the map coordinate system and an IfcMapConversion or `IfcRigidOperation` that defines the mapping between `IfcProjectedCRS` and `IfcGeometricRepresentationContext`. 
+- (RI-87) The model shall containts an `IfcProjectedCRS` that defines the map coordinate system and an `IfcMapConversion` or `IfcRigidOperation` that defines the mapping between `IfcProjectedCRS` and `IfcGeometricRepresentationContext`. 
```

Comment:
@apinzenoehler, this was suggested as a very good fit for a Gherkin rule (GRF002). Are you going to leave it as textual requirement or are you still planning to develop a rule for it?

Reply:
According to my memory this was already discussed between @aothms and @pjanck and is work in progress.

@aothms and @pjanck : is my memory correct?

Reply:
Your memory is correct. I believe the final conclusion was, that this requirement imposes no additional requirements on the dataset as what is already defined by the schema.

We are, however, imposing a test plan specific requirement of the specific CRS in use - which was deemed out-of-scope for a Gherkin rule.